terraform/setup/setup.tf

```diff
@@ -16,6 +16,19 @@
 # in this module, we create the necessary common resources for the integ-tests, this setup module will only need to be executed once.
 # vpc, iam role, security group, the number of those resources could be limited, creating them concurrently for every pr would trigger throttling issue.
 
+#(TODO) pvasir - The terraform major version 4.0.0 is causing an versioning issue for s3 bucket, the version shall be updated once the fix is availble
+# reference - https://github.com/hashicorp/terraform-provider-aws/issues/23106
+/*
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "~> 3.74"
```

Comment:
Why are we commenting this out rather than pinning to a version prior to 4.0.0? When did our canary tests start to pick up version 4.0.0?

Reply:
Also why is this comment out rather than use the solution provided in your link [here ](https://github.com/hashicorp/terraform-provider-aws/issues/23125#issuecomment-1038424883)

Reply:
sure, probably, can create a seperate PR if this is required. since there is a workaround shall remove the comment. thank you

Reply:
Yea, if it is not relevant to this PR it should probably be removed